semaphore/templates/brakeman.yml

```diff
@@ -15,10 +15,6 @@ global_job_config:
   secrets:
     - name: global-secrets
 
-  env_vars:
-    - name: BRAKEMAN_REPORT_HTML
-      value: brakeman.html
-
```

Comment:
I'm just hardcoding it to `brakeman.html` now because we can easily control it from this centralized location.